### PR TITLE
vebt-851 FE - 22-10215 - Confirmation

### PR DESCRIPTION
**Pull request by GovNapoleon**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

If the folder you changed contains a `manifest.json`, search for its `entryName` in the content-build [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) (the `entryName` there will match).

If an entry for this folder exists in content-build and you are:
1. **Deleting a folder**:
   1. First search `vets-website` for _all_ instances of the `entryName` in your `manifest.json` and remove them in a separate PR. Look particularly for references in `src/applications/static-pages/static-pages-entry.js` and `src/platform/forms/constants.js`. _**If you do not do this, other applications will break!**_
      - _Add the link to your merged vets-website PR here_
   2. Then, Delete the application entry in [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) and merge that PR **before** this one
      - _Add the link to your merged content-build PR here_

2. **Renaming or moving a folder**: Update the entry in the [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json), but do not merge it until your vets-website changes here are merged. The content-build PR must be merged immediately after your vets-website change is merged in to avoid CI errors with content-build (and Tugboat).

### :warning: TeamSites :warning:
Examples of a TeamSite: https://va.gov/health and https://benefits.va.gov/benefits/. This scenario is also referred to as the "injected" header and footer. You can reach out in the `#sitewide-public-websites` Slack channel for questions.

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

-  VEBT-851 FE - 22-10215 - Confirmation
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Jira

As a...

VEBT Developer 

I want…

To build the final "step" of the 22-10215

So that…

Form development can be completed

Acceptance Criteria

NOTE:- This ticket has been updated to [VEBT-849](https://jira.devops.va.gov/browse/VEBT-849) due to recent design changes and consolidation. Effort done by dev of this ticket still stands.

Once PDF is generated and downloaded, a new page will appear with next steps and info.

h1 -> Report 85/15 Rule enrollment ratios

Statement of Assurance of Compliance with 85% Enrollment Ratios (VA Form 22-10215)

To submit your form, follow the steps below
1
Download and save your form
Make sure that your completed form is saved as a PDF on your device. If you didn’t do that on the previous page, go back and do that now. 
2
Upload the form to the VA education portal
Visit the VA Education File Upload Portal (opens in a new tab), and upload your saved VA Form 22-10215.
3
Submit your form
Once uploaded, click submit to finalize your request.              
Back
What are my next steps?
After you submit your 85/15 Rule enrollment ratios, we will review them within 7-10 business days. Once we complete the review, we will email your school a letter with the decision. If we accept your request, we will include a copy of WEAMS form 1998 as confirmation in the letter. If we deny your request, we will explain the reason for rejection in the letter and provide further instructions for re-submission or additional steps

Other details :- 

Technical Details:

Dependencies: 

Feature Flag needed: Yes

Demo needed: No

UAT needed: Yes

Additional Notes:  

Testing Details:

(  ) Quick Turn Around/Direct to Production

( X ) Remains in Staging

(  ) Staging Environment unavailable, coordinate w/ developer

Requested by and when:

Cinda, Forms creation

Definition of Done: 

(  )  Demo/UAT completed with EDU

(  )  Passed all internal testing in Staging

(  )  Passed all internal testing in Production

(  )  All work documented in Jira ticket
## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#0000
- _Link to previous change of the code/bug (if applicable)_
department-of-veterans-affairs/vets-website#0000
- _Link to epic if not included in ticket_
department-of-veterans-affairs/va.gov-team#0000

## Testing done

- Verified changes on the local machine
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

vebt-851-local

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/edu-benefits/10215/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -1,69 +1,115 @@
...
+          <p>Once uploaded, click submit to finalize your request.</p>
+        </div>
+      </va-process-list-item>
+    </va-process-list>
+    <p>
+      <va-button
+        secondary
+        text="Print this page"
+        onClick={() => window.print()}
+        data-testid="ezr-print-button"
```
Comment (GovCIOLiz) [inline src/applications/edu-benefits/10215/containers/ConfirmationPage.jsx:76]: Can remove

File: src/applications/edu-benefits/10215/containers/ConfirmationPage.jsx [hunk 2]
```diff
@@ -1,69 +1,115 @@
...
+        This form requires additional steps for successful submission. Follow
+        the instructions below carefully to ensure your form is submitted
+        correctly.
+      </p>
+    </va-alert>
+    <h2 className="vads-u-font-size--h3 vad-u-margin-top--0">
+      To submit your form, follow the steps below
+    </h2>
+    <va-process-list uswds>
+      <va-process-list-item header="Check your eligibility">
```
Comment (GovCIOLiz) [inline src/applications/edu-benefits/10215/containers/ConfirmationPage.jsx:28]: All of the headers shown in the screenshot say "Check your eligibility" vs the titles shown in design

File: src/applications/edu-benefits/10215/containers/ConfirmationPage.jsx (hunk 1 above)
Comment (GcioNapoleon) [reply src/applications/edu-benefits/10215/containers/ConfirmationPage.jsx:76]: Can you please specify which part you want me to remove. Do you want me to remove all 4 lines are just the last line.

File: src/applications/edu-benefits/10215/containers/ConfirmationPage.jsx (hunk 2 above)
Comment (GcioNapoleon) [reply src/applications/edu-benefits/10215/containers/ConfirmationPage.jsx:28]: Changing now

Comment (GcioNapoleon) [reply src/applications/edu-benefits/10215/containers/ConfirmationPage.jsx]: Done

File: src/applications/edu-benefits/10215/containers/ConfirmationPage.jsx (hunk 1 above)
Comment (GcioNapoleon) [reply src/applications/edu-benefits/10215/containers/ConfirmationPage.jsx]: Done

**Review by GovCIOLiz — APPROVED** [review]

**Pull request merged**